content/network/drivers/overlay.md

```diff
@@ -40,23 +40,21 @@ apply to overlay networks used by standalone containers.
 
 ### Create an overlay network
 
-> **Prerequisites**:
```

Comment:
Took this content out of a callout since this was a lot of real estate for a callout box.